<launch>
    <!-- 启动底层驱动 Start the underlying driver -->
    <include file="$(find yahboomcar_bringup)/launch/bringup.launch"/>
    <!-- 启动激光雷达 Start lidar -->
    <include file="$(find rplidar_ros)/launch/rplidar.launch"/>
    <!-- 启动巡逻节点 Start patrol node -->
    <node pkg="yahboomcar_bringup" type="patrol.py" name="YahboomCarPatrol" required="true" output="screen">
        <param name="circle_adjust" type="double" value="2.0"/>
    </node>
</launch>
